BeagleBone_black/src/communications/main.cpp

```diff
@@ -153,220 +155,52 @@ int Main::sendTorqueBl(float torquebl)
-{
-  if (operational) {
-    return baseCommunicator_->sendData("CMD3501\n");
-  }
-
-  return baseCommunicator_->sendData("CMD3500\n");
-}
-
-int Main::sendProxiFront8(bool operational)
-{
-  if (operational) {
-    return baseCommunicator_->sendData("CMD3601\n");
-  }
-
-  return baseCommunicator_->sendData("CMD3600\n");
-}
-
-int Main::sendProxiRear1(bool operational)
-{
-  if (operational) {
-    return baseCommunicator_->sendData("CMD3701\n");
-  }
-
-  return baseCommunicator_->sendData("CMD3700\n");
-}
-
-int Main::sendProxiRear2(bool operational)
-{
-  if (operational) {
-    return baseCommunicator_->sendData("CMD3801\n");
-  }
-
-  return baseCommunicator_->sendData("CMD3800\n");
-}
-
-int Main::sendProxiRear3(bool operational)
-{
-  if (operational) {
-    return baseCommunicator_->sendData("CMD3901\n");
-  }
-
-  return baseCommunicator_->sendData("CMD3900\n");
-}
-
-int Main::sendProxiRear4(bool operational)
-{
-  if (operational) {
-    return baseCommunicator_->sendData("CMD4001\n");
-  }
-
-  return baseCommunicator_->sendData("CMD4000\n");
-}
-
-int Main::sendProxiRear5(bool operational)
-{
-  if (operational) {
-    return baseCommunicator_->sendData("CMD4101\n");
-  }
-
-  return baseCommunicator_->sendData("CMD4100\n");
-}
-
-int Main::sendProxiRear6(bool operational)
-{
-  if (operational) {
-    return baseCommunicator_->sendData("CMD4201\n");
-  }
-
-  return baseCommunicator_->sendData("CMD4200\n");
-}
-
-int Main::sendProxiRear7(bool operational)
-{
-  if (operational) {
-    return baseCommunicator_->sendData("CMD4301\n");
-  }
-
-  return baseCommunicator_->sendData("CMD4300\n");
-}
-
-int Main::sendProxiRear8(bool operational)
-{
-  if (operational) {
-    return baseCommunicator_->sendData("CMD4401\n");
-  }
-
-  return baseCommunicator_->sendData("CMD4400\n");
+int Main::sendImu(bool operational, bool operational1, bool operational2,
+bool operational3, bool operational4, bool operational5, bool operational6, bool operational7)
+{
+  std::string sns1, sns2, sns3, sns4, sns5, sns6, sns7, sns8;
+  if (operational) {sns1 = "1";} else {sns1 = "2";}
+  if (operational1) {sns2 = "1";} else {sns2 = "2";}
+  if (operational2) {sns3 = "1";} else {sns3 = "2";}
+  if (operational3) {sns4 = "1";} else {sns4 = "2";}
+  if (operational4) {sns5 = "1";} else {sns5 = "2";}
+  if (operational5) {sns6 = "1";} else {sns6 = "2";}
+  if (operational6) {sns7 = "1";} else {sns7 = "2";}
+  if (operational7) {sns8 = "1";} else {sns8 = "2";}
+  return baseCommunicator_->sendData("CMD21" + sns1 + sns2 +
+  sns3 + sns4 + sns5 + sns6 + sns7 + sns8 + "\n");
+}
+
+int Main::sendProxiFront(bool operational, bool operational1, bool operational2,
+bool operational3, bool operational4, bool operational5, bool operational6, bool operational7)
+{
+  std::string sns1, sns2, sns3, sns4, sns5, sns6, sns7, sns8;
+  if (operational) {sns1 = "1";} else {sns1 = "2";}
+  if (operational1) {sns2 = "1";} else {sns2 = "2";}
```

Comment:
1. operational1 but sns2? would be more consistent if operational1 and sns1.

2. it might be neater to do 
`int sns = operational ? 1 : 2;`